File: problemas/iniciante/1589/1589.c
#include <stdio.h>

int main()
{
    int T, R1, R2;

    scanf("%d", &T);
    for (int i = 0; i < T; ++i)
    {
        scanf("%d %d", &R1, &R2);
        printf("%d\n", R1 + R2);
    }

    return 0;
}

File: problemas/iniciante/1589/1589.go
package main

import (
    "fmt"
)

func main() {
    var n, r1, r2 int

    fmt.Scanf("%d", &n)
    for i := 0; i < n; i++ {
        fmt.Scanf("%d %d", &r1, &r2)
        fmt.Printf("%d\n", r1+r2)
    }
}
